.github/workflows/ci.yml

```diff
@@ -22,19 +22,21 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.12.15, 2.13.7]
-        java: [adopt@1.8]
+        scala: [2.12.16, 2.13.10]
+        java: [adopt-hotspot@1.8]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout current branch (full)
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Setup Java and Scala
-        uses: olafurpg/setup-scala@v13
+      - name: Setup Java (adopt-hotspot@1.8)
+        if: matrix.java == 'adopt-hotspot@1.8'
+        uses: actions/setup-java@v2
         with:
-          java-version: ${{ matrix.java }}
+          distribution: adopt-hotspot
+          java-version: 8
```

Comment:
[Could not find satisfied version for SemVer '1.8'](https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md#dropping-legacy-java-version-syntax-1x)

Reply:
`sbt githubWorkflowGenerate`

Reply:
should be what you run....